index.html

```diff
@@ -2630,8 +2630,8 @@ <h2>Definition of Roles</h2>
 		<div class="role" id="complementary">
 			<rdef>complementary</rdef>
 			<div class="role-description">
-				<p>A <rref>landmark</rref> that is designed to be complementary to the main content at a similar level in the DOM hierarchy, but remaining meaningful when separated from the main content.</p>
-				<p>There are various types of content that would appropriately have this <a>role</a>. For example, in the case of a portal, this may include but not be limited to show times, current weather, related articles, or stocks to watch. The complementary role indicates that contained content is relevant to the main content. If the complementary content is completely separable from the main content, it may be appropriate to use a more general role.</p>
+				<p>A <rref>landmark</rref> that is designed to be complementary to the main content it is a direct descendant of, or is at a similar level in the DOM hierarchy. The contents of a complementary landmark would be expected to remain meaningful if it were to be separated from the main content it is relevant to.</p>
```

Comment:
```suggestion
				<p>A <rref>landmark</rref> that is designed to be complementary to the main content it is a sibling to, or a direct descendant of. The contents of a complementary landmark would be expected to remain meaningful if it were to be separated from the main content it is relevant to.</p>
```

thoughts?

Reply:
Add a second "that" for readability: `A landmark that is designed to be complementary to the main content that it is a sibling to, or a direct descendant of.`
other than that looks good!